JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -1,12 +1,31 @@
 //
 //  JuiceMaker - FruitStore.swift
-//  Created by yagom. 
+//  Created by dasan & kyungmin.
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
-
-// 과일 저장소 타입
 class FruitStore {
+    private(set) var fruitInventory: [FruitType: Int] = [.strawberry: 10, .banana: 10, .pineapple: 10, .kiwi: 10, .mango: 10]
+    
+    func checkStock(fruit: FruitType, amount: Int) -> Bool {
+        guard let fruitStock = fruitInventory[fruit],
+              fruitStock >= amount else {
+            return false
+        }
+        return true
+    }
+    
+    func addStock(fruit: FruitType, addCount: Int) {
```

Comment:
addStock이라는 메서드명에서 추가한다는 의미를 충분히 전달하고 있어서 매개변수 명이 addCount일 필요는 없을거 같네요!
```suggestion
    func addStock(fruit: FruitType, count: Int) {
```